.github/workflows/translations.yml

```
@@ -0,0 +1,51 @@
+name: Check translations
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v1
```

v1 is quite old. Use newest version.

```suggestion
        uses: actions/setup-python@v4
```